source/manuals/code-review-guidelines.html.md.erb

```diff
@@ -90,16 +90,15 @@ If you want to learn more about writing clearly for technical audiences you can
 
 ## Further reading
 
-Find out more about writing and reviewing pull requests on the [GDS Tech Learning Pathway][].
+Find out more about writing and reviewing pull requests on the [Pull requests page](/standards/pull-requests.html).
 
 [GitHub review]: https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/about-pull-request-reviews
 [pull requests (PRs)]: https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests
-[technical debt]: https://gds-way.cloudapps.digital/standards/technical-debt.html
-[programming language style guides]: https://gds-way.cloudapps.digital/manuals/programming-languages.html
-[open source]: https://gds-way.cloudapps.digital/standards/publish-opensource-code.html
-[how to manage third party software dependencies here]: https://gds-way.cloudapps.digital/standards/tracking-dependencies.html
+[technical debt]: /standards/technical-debt.html
+[programming language style guides]: /manuals/programming-languages.html
+[open source]: /standards/publish-opensource-code.html
+[how to manage third party software dependencies here]: /standards/tracking-dependencies.html
 [deploying software]: https://www.gov.uk/service-manual/technology/deploying-software-regularly
-[GDS Tech Learning Pathway]: https://github.com/alphagov/gds-tech-learning-pathway/blob/main/resources/other/code-reviews.html.md
-[Programming language style guides]: https://gds-way.cloudapps.digital/manuals/programming-languages.html
+[Programming language style guides]: /manuals/programming-languages.html
```

Comment:
Very minor but it looks like `[Programming language style guides]` is now duplicated with two entries. 

Even more minor but all of the other links on the page include the link at the bottom and then reference in the link in the markdown above, consistency with this pattern for the new `[Pull requests page]` might make the file easier to scan

Reply:
Thanks SF. You folks are all the best